bigtable/client/client_options.h

```diff
@@ -92,6 +92,27 @@ class ClientOptions {
     channel_arguments_.SetGrpclbFallbackTimeout(fallback_timeout);
   }
 
+  /**
+   * Set the grpclb fallback timeout (std::chrono::duration<> duration) for the
+   * channel.
+   *
+   * Please see the docs for grpc::ChannelArguments::SetGrpclbFallbackTimeout()
+   * on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html
+   * for more details.
+   *
+   */
+  template <typename Rep, typename Period>
+  void SetGrpclbFallbackTimeout(
+      std::chrono::duration<Rep, Period> fallback_timeout) {
+    int fallback_timeout_ms = 0;
+
+    std::chrono::milliseconds ft_ms =
+        std::chrono::duration_cast<std::chrono::milliseconds>(fallback_timeout);
+    fallback_timeout_ms = static_cast<std::int32_t>(ft_ms.count());
```

Comment:
What if `ft_ms.count()` is larger than what fits in a `std::int32_t`?  The documentation should say what happens and we should raise an exception (probably `std::out_of_range`).